opencti-platform/opencti-front/lang/front/fr.json

```diff
@@ -528,6 +528,7 @@
   "dashboard": "Tableau de bord",
   "Dashboard settings": "Paramètres du tableau de bord",
   "Dashboards": "Tableaux de bord",
+  "Dashboards from your groups & organizations": "Tableaux de bord de vos groupes & ; organisations",
```

Comment:
why "& ;" in this translation?

Reply:
How, didn't see that, deepl doesn't like & it seems :D I will update the translations